src/vma/sock/sockinfo_tcp.cpp

```diff
@@ -2350,7 +2345,6 @@ int sockinfo_tcp::listen(int backlog)
 			m_p_rx_ring = rx_ring_iter->first;
 		}
 		si_tcp_logdbg("sock state = %d", get_tcp_state(&m_pcb));
-		success = true;			
```

Comment:
@igor-ivanov why did you drop this assignment? 
```
success = true;
```
it seems like changing the flow logic